src/character_effects.cpp

```diff
@@ -260,7 +260,7 @@ int calc_focus_change( const Character &who )
 
     // for every 100 points, we have a flat gain of 1 focus.
```

Comment:
```suggestion
    // for every 10 points, we have a flat gain of 1 focus.
```

docstring isn't updated.